packages/Algernon.package/ALGNamedWorldMenuEntry.class/instance/enabledMenuSubmorphs.st

```diff
@@ -2,4 +2,4 @@ accessing
 enabledMenuSubmorphs
 	
 	"Some submorphs can't be disabled (e.g. line, ...)"
-	^ self menuSubmorphs select: [:menuItem | (menuItem class canUnderstand: #isEnabled) not or: [menuItem isEnabled]]
\ No newline at end of file
+	^ (self menuSubmorphs, self subMenuSubmorphs) select: [:menuItem | (menuItem class canUnderstand: #isEnabled) not or: [menuItem isEnabled]]
```

Comment:
Do you mean `{ a . b }` to create an array? The `,` is string concat, right?

Reply:
I think this method is not even used anymore, is it? At least we removed the `subMenuSubmorphs` method...

Reply:
That is ... interesting. CI would have (hopefully) caught that aswell

Reply:
For ordered collections, the `,` merges them (instead of creating a 2D array), so I think it is correct.
If the method is not used anymore, we probably should remove it anyway :D

Reply:
ohhh, yeah that actually makes sense, well then forget that comment